README.md

```diff
@@ -40,6 +40,8 @@ If you are looking to make your first code contribution to this project then we
 - [Motion Sensor](https://www.aqara.com/en/motion_sensor.html): lumi.sensor_motion.aq2
 - [Temperature / Humidity Sensor](https://www.aqara.com/en/temperature_and_humidity_sensor-product.html): lumi.weather
 - [Water Leak](https://www.aqara.com/en/water_leak_sensor.html): lumi.sensor_wleak.aq1
+- [US Plug](https://www.aqara.com/en/smart_plug.html): lumi.plug.maus01
+- [EU Plug](https://zigbee.blakadder.com/Xiaomi_ZNCZ04LM.html): lumi.plug.mmeu01
```

Comment:
I could not find an official page for this particular model

Reply:
np